_layouts/base.html

```diff
@@ -3,17 +3,16 @@
 <!--
     ============================================================================
 
-    Hey! If you're viewing this, you should probably come work on our Technology
-    & Innovation team. We're always looking for a few great designers,
-    developers, data scientists, and network, infrastructure, privacy and
-    security pros. Keep an eye on our job opportunities at:
-    http://www.consumerfinance.gov/jobs/
+    Hey! If you're viewing this, you should probably come work on our
+    Technology & Innovation team. We're always looking for a few great
+    designers; developers; data scientists; and network, infrastructure,
```

Comment:
As @ascott1 points out, I didn't write it, I just punctuated it properly. I agree that it's a mouthful.

If someone wants to pitch new language, I'll happily revise. I'd rather not make it a one-item-per-line-formatted-list, though, because then this comment that's already 15 lines tall will grow even larger.